### DISCUSSION TITLE
Kto jest odpowiedzialny za to, żeby gałąź (branch) była aktualna (up to date) względem gałęzi (branch) w3c/wcag:main?

### DISCUSSION THREAD

**Discussion by speplinski** [irdpl/wcag#7] [Organizacja pracy]
W pracy z gałęziami (branchami), gdy mówimy o tym, że dana gałąź jest aktualna, to oznacza, że zawiera najnowsze zmiany i jest zsynchronizowana z innymi gałęziami lub źródłem, z którym chcemy ją porównać.

Utrzymywanie gałęzi w synchornizacji z głównym kodem jest ważne, aby uniknąć problemów z integracją kodu i zachować porządek i przejrzystość w zarządzaniu kodem.

Z tego też względu wydaje się bardzo ostatnie ustalić procedurę synchronizacji i osobę/osoby za to odpowiedzialne.

**Comment by zwiastunsw** [comment]
1. W3C najprawdopodobniej nie wszystko posyła jako PR, więc synchronizacja może być trudna.
2. Dobrze byłoby opracować takiego bota, który by np. zakładał issues zgłaszający, że w oryginalnej wersji nastąpiła zmiana, która wymaga działania.
3. W razie co mamy gdzie indziej takiego bocika do podglądu.
